docs/elements.md

```diff
@@ -509,6 +509,7 @@ Given a list of statements, select a matching option for each entry from a drop-
 | `blank`               | boolean                                                    | True          | Option to add blank dropdown entry as the default selection in each drop-down list.                                                                                                                                    |
 | `counter-type`        | "decimal" or "lower-alpha" or "upper-alpha" or "full-text" | "lower-alpha" | The type of counter to use when enumerating the options. If set to "full-text", the column of options will be hidden, and the text of each option will be used in the statements' dropdown lists, instead of counters. |
 | `hide-score-badge`    | boolean                                                    | false         | Whether or not to hide the correct/incorrect score badge next to each graded answer choice.                                                                                                                            |
+| `options-placement`   | string                                                     | "right"       | The placement of options relative to the statements in order to make it visually cohesive. Especially useful when dealing with long statements or options.                                                             |
```

Comment:
Aren't the only available options "right" or "bottom"? You should list these instead of putting string as the type.

Reply:
Oh okay, noted!